data/rubyday/rubyday-2011/sponsors.yml

```diff
@@ -0,0 +1,52 @@
+---
+- tiers:
+    - name: Gold
+      description: "Gold sponsors (€1,000 contribution). Companies receive visibility on the website, public acknowledgments at the start and end of the event, ability to collect participant CVs, and ability to distribute promotional materials during the event."
+      level: 1
+      sponsors:
+        - name: Mikamai
+          website: https://mikamai.com/
+          slug: mikamai
+          logo_url: ""
```

Comment:
@adrienpoly @marcoroth is the logo required? Because it usually references the URL on the site, but all sites are gone (only accessible via web archive).

It also used to be based on a sprite image (yay 2010!).